packages/compass-collection-stats/src/stores/store.js

```diff
@@ -5,266 +5,145 @@ import numeral from 'numeral';
 import createDebug from 'debug';
 const debug = createDebug('compass-collection-stats:store');
 
+const kInstance = Symbol('instance');
```

Comment:
I guess it would be enough to add a comment, I started to think weird stuff, like, if is a symbol then reflux won't add a watcher etc .. 🤣

if we want a simpler way probably just `store._appRegistry` and `store._instance` 🤷‍♂️, but i'm ok with the symbol

Reply:
I'll keep it but will add a comment! Thanks for highlighting this

Reply:
Done in 9b8a800